manifests/init.pp

```diff
@@ -112,10 +124,9 @@
     minute  => 10,
   }
 
-  service { 'spamassassin':
+  service { $sa_service:
     ensure  => $service_ensure,
     enable  => $service_enable,
     require => Package['spamassassin'],
-    pattern => 'spamd',
```

Comment:
I don't love seeing this removed, but I don't see where it was actually used.  The debian init script (at least) provides a status command...